tiatoolbox/tools/patchextraction.py

```diff
@@ -54,14 +62,22 @@ class PatchExtractor(ABC):
         pad_constant_values (int or tuple(int)): Values to use with
           constant padding. Defaults to 0. See :func:`numpy.pad` for
           more.
+        within_bound (bool): whether to extract patches beyond the
+          input_image size limits. If yes, extracted patches at margins
+          will be padded appropriately based on `pad_constant_values` and
+          `pad_mode`. Default is False.
 
     Attributes:
         wsi(WSIReader): input image for patch extraction of type :obj:`WSIReader`.
         patch_size(tuple(int)): patch size tuple (width, height).
         resolution(tuple(int)): resolution at which to read the image.
         units (str): the units of resolution.
         n(int): current state of the iterator.
-        locations_df(pd.DataFrame): A table containing location and/or type of patch.
+        locations_df(pd.DataFrame): A table containing location and/or type of patces
+          in `(x_start, y_start, class)` format.
+        coord_list(:class:`numpy.ndarray`): An array containing coordinates of patches
```

Comment:
why we need this separately?

Reply:
`locations_df` contains only starting `x,y` positions while `coord_list` bounding box information to be used for read bound. I kept both at the moment as `locations_df` might have been used somewhere else.